project/LinkerdBuild.scala

```diff
@@ -265,31 +265,28 @@ object LinkerdBuild extends Base {
          |"""
       ).stripMargin
 
-    val Minimal = config("minimal")
-    val MinimalSettings = Defaults.configSettings ++ appPackagingSettings ++ Seq(
+    val BundleSettings = Defaults.configSettings ++ appPackagingSettings ++ Seq(
       mainClass := Some("io.buoyant.namerd.Main"),
       assemblyExecScript := execScript.split("\n").toSeq,
       dockerEnvPrefix := "NAMERD_",
-      unmanagedBase := baseDirectory.value / "plugins"
-    )
-
-    val MinimalProjects = Seq[ProjectReference](
-      core, main, Namer.fs, Storage.inMemory, Router.http,
-      Iface.controlHttp, Iface.interpreterThrift
-    )
-
-    val Bundle = config("bundle") extend Minimal
-    val BundleSettings = MinimalSettings ++ Seq(
+      unmanagedBase := baseDirectory.value / "plugins",
       assemblyJarName in assembly := s"${name.value}-${version.value}-exec",
       dockerTag := version.value
     )
 
     val BundleProjects = Seq[ProjectReference](
+      core, main, Namer.fs, Storage.inMemory, Router.http,
+      Iface.controlHttp, Iface.interpreterThrift,
       Namer.consul, Namer.k8s, Namer.marathon, Namer.serversets, Namer.zkLeader,
       Interpreter.perHost, Interpreter.k8s,
       Storage.etcd, Storage.inMemory, Storage.k8s, Storage.zk, Storage.consul
     )
 
+    val LowMemSettings = BundleSettings ++ Seq(
+      dockerJavaImage := "buoyantio/debian-32-bit",
+      dockerTag := s"${version.value}-32bit"
+    )
```

Comment:
We should ensure that LowMemSettings uses the assemblyJarName defined in appPackagingSettings and not that defined in BundleSettings.

Furthermore, would prefer 32b to 32bit